lib/v2/Dropdown.js

```diff
@@ -1,11 +1,11 @@
-import React, { useState, useRef, useEffect } from "react";
+import React, { useState, useEffect } from "react";
 import { usePopper } from "react-popper";
 import Button from "./Button";
 import { Down } from "@bigbinary/neeto-icons";
 import classnames from "classnames";
 import { hyphenize } from "../common";
 import { useHotkeys } from "react-hotkeys-hook";
-import mergeRefs from "merge-refs";
+import OutsideClickHandler from "react-outside-click-handler";
```

Comment:
@amaldinesh7 _a 
Please sort the imports. Package import should on the top. Ref https://www.bigbinary.com/react-best-practices/sort-import-statements#sorting-import-statements

Please make this change and merge the PR. Rest looks good.